package.json

```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "preversion": "yarn build-all",
     "bootstrap": "yarn prep",
-    "prep": "yarn lerna bootstrap",
+    "prep": "yarn && yarn lerna bootstrap && cd packages/react-cool-storage",
```

Comment:
What is this `cd packages/react-cool-storage` for?

Reply:
Removed this